composer.json

```diff
@@ -31,7 +30,8 @@
     "spryker/payone": "*"
   },
   "suggest": {
-    "spryker/checkout": "If you want to use Checkout plugins you need to install spryker/checkout."
+    "spryker/checkout": "If you want to use Checkout plugins you need to install spryker/checkout.",
+    "spryker/step-engine": "If you want to use StepEngine plugins you need to install spryker/step-engine."
```

Comment:
If sth is in suggest it should also be in require-dev.

Reply:
Fixed.